docs/website/docs/examples/qdrant_zendesk/code/qdrant-snippets.py

```diff
@@ -0,0 +1,191 @@
+
+
+def qdrant_snippet():
```

Comment:
please prevent this test from running on forks, we have a special pytest marker for that